src/olympia/constants/activity.py

```diff
@@ -868,6 +868,15 @@ class LOG_IN_API_TOKEN(_LOG):
     format = '{user_responsible} authenticated through an API token.'
 
 
+class CLEAR_NEEDS_HUMAN_REVIEWS(_LOG):
+    id = 173
+    format = _('{addon} {version} needs_human_review flag cleared.')
```

Comment:
Technically the `version` would be the latest in the channel because we're always passing one from `log_action`... unless the add-on doesn't have any.

That does seem wrong/problematic. We probably should only pass the `addon` to the `ActivityLog` and don't use `version` in the template.

Reply:
Note, if we don't pass a version we lose the ability to know which channel had it's flag's cleared.  Maybe we don't care about that though. Or we could specifically add `channel` to the details JSON blob that gets stored too?

Reply:
I'm not sure we care that much. Maybe having the `channel` in the details JSON blob is a good insurance policy yeah.

Reply:
https://github.com/mozilla/addons-server/pull/20386/commits/5379146e5c70d0be4759578115f308e913e0d22b